TheInternetTestSuit/DropdownTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Dropdown;
+using Welcome;
+
+namespace TheInternetTestSuit
+{
+    [TestFixture]
+    public class DropdownTests : BaseTest
+    {
+        private WelcomePage welcomePage;
+        private DropdownPage dropdownPage;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            Driver.Navigate().GoToUrl(MainUrl);
+            welcomePage = new WelcomePage(Driver);
+            dropdownPage = new DropdownPage(Driver);
+            welcomePage.DisplayDropdownPage();
+            Assert.That(Driver.Url, Is.EqualTo($"{MainUrl}/dropdown"), "Dropdown page is not displayed");
+        }
+
+        [Test]
+        public void SelectOption()
+        {
+            Assert.That(dropdownPage.SelectOption("Option 1"), Is.EqualTo("Option 1"), "Selected option is not 'Option 1'");
+        }
+
+        [Test]
+        public void SelectDifferentOption()
+        {
+            SelectOption();
```

Comment:
it's better to create a separate private method below, do not run separate test in another one

Reply:
Ok, done that